### ISSUE TITLE
Allow a range of ages in AgeShape

### ISSUE THREAD

**Issue by apdavison** [INCF/neuroshapes#72]
(as defined in `core/subject`)

AgeShape requires a single float value, but some datasets only specify a range of ages, e.g. "21-30 days", rather than providing the precise age of each subject.

**Comment by MFSY** [comment]
Good point. [schema:maxValue](http://schema.org/maxValue) and [schema:minValue](http://schema.org/minValue) can be used to represent range in quantitative value (which is the shape of age value).
